core/src/main/resources/defaults/blueprints/7.2.11/cdp-flow-management-small.bp

```diff
@@ -3,6 +3,7 @@
   "blueprint": {
     "cdhVersion": "7.2.11",
     "displayName": "flow-management",
+    "blueprintUpgradeOption": "OS_UPGRADE_DISABLED",
```

Comment:
For 7.2.11 this should be allowed. Anything that is below this version must be disabled.